src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -198,14 +198,10 @@ public void OnStartActivity(Activity activity, object payload)
                 activity.DisplayName = path;
 
                 // see the spec https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md
-
-                if (request.Host.Port is null or 80 or 443)
-                {
-                    activity.SetTag(SemanticConventions.AttributeHttpHost, request.Host.Host);
-                }
-                else
+                activity.SetTag(SemanticConventions.AttributeNetHostName, request.Host.Host);
```

Comment:
need any check to confirm if Host is set?

Reply:
not sure, will have to check.

Reply:
added a check